cmd/osdAddonRelease.go

```diff
@@ -221,8 +198,9 @@ func newOSDAddonReleseCmd(flags *osdAddonReleaseFlags, gitlabToken string) (*osd
 	}
 	fmt.Print("gitlab client initialized and authenticated\n")
 
+	gitCloneService := &services.DefaultGitCloneService{}
 	// Clone the managed tenants
-	managedTenatsDir, managedTenantsRepo, err := gitCloneToTmp(
+	managedTenatsDir, managedTenantsRepo, err := gitCloneService.CloneToTmpDir(
```

Comment:
```suggestion
	// TODO: Move the clone functions inise the run() method to improve the test covered code
	managedTenatsDir, managedTenantsRepo, err := gitCloneService.CloneToTmpDir(
```
I would like to add this TODO so the next time we put hands on the osdAddonRelease we can do it